### PR TITLE
Add note regarding redeploying metal3-dev-env

### PR DESCRIPTION
**Pull request by furkatgofurov7**
Fixes [metal3-dev-env/issues/434](https://github.com/metal3-io/metal3-dev-env/issues/434)

### CODE REVIEW

**Comment by furkatgofurov7** [comment]
/assign @dhellmann
/cc @maelk
/cc @fmuyassarov 

Is this failing because of #188 ?

File: try-it.md [hunk 1]
```diff
@@ -76,6 +76,11 @@ To tear down the environment, run
 $ make clean
 ```
 
+> info "Note"
+> in case of redeploying metal³-dev-env with a different release version of
```
Comment (dhellmann) [inline try-it.md:80]:
```suggestion
> When redeploying metal³-dev-env with a different release version of
```

File: try-it.md [hunk 2]
```diff
@@ -76,6 +76,11 @@ To tear down the environment, run
...
+> CAPM3 requires `FORCE_REPO_UPDATE` variable in `config_${user}.sh` set to
```
Comment (dhellmann) [inline try-it.md:81]:
```suggestion
> CAPM3, you must set the `FORCE_REPO_UPDATE` variable in `config_${user}.sh` set to
```

**Review by fmuyassarov — COMMENTED** [review]
Travis failed because https://github.com/metal3-io/metal3-io.github.io/pull/188 is not merged yet. There are other two PRs https://github.com/metal3-io/metal3-io.github.io/pull/185 and https://github.com/metal3-io/metal3-io.github.io/pull/186 failed due to the same reason.
@dhellmann can we proceed with merging https://github.com/metal3-io/metal3-io.github.io/pull/185 if it lgty ? ( correction https://github.com/metal3-io/metal3-io.github.io/pull/188)

**Comment by furkatgofurov7** [comment]
> Travis failed because #188 is not merged yet. There are other two PRs #185 and #186 failed due to the same reason.
> @dhellmann can we proceed with merging #185 if it lgty ?

@fmuyassarov you mean #188 maybe?

**Comment by fmuyassarov** [comment]
> > Travis failed because #188 is not merged yet. There are other two PRs #185 and #186 failed due to the same reason.
> > @dhellmann can we proceed with merging #185 if it lgty ?
> 
> @fmuyassarov you mean #188 maybe?

yes sorry, #188

File: try-it.md (hunk 1 above)
Comment (furkatgofurov7) [reply try-it.md]: Thanks, updated.

File: try-it.md (hunk 2 above)
Comment (furkatgofurov7) [reply try-it.md]: Thanks, updated.

**Comment by furkatgofurov7** [comment]
@dhellmann this was approved earlier, but travis was failing and #188 fixed it, would you mind approving it again? Thanks!
/cc @russellb

**Pull request merged**